src/index.js

```diff
@@ -248,4 +252,41 @@ IdTokenVerifier.prototype.decode = function (token) {
   };
 };
 
+/**
+ * @callback validateAccessTokenCallback
+ * @param {Error} [err] error returned if the validation cannot be performed
+ * or the token is invalid. If there is no error, then the access_token is valid.
+ */
+
+/**
+ * Validates an access token based on {@link http://openid.net/specs/openid-connect-core-1_0.html#ImplicitTokenValidation}
+ *
+ * @method validateAccessToken
+ * @param {String} access_token the access_token
+ * @param {String} id_token the id_token
+ * @param {validateAccessTokenCallback} cb callback used to notify the results of the validation.
+ */
+IdTokenVerifier.prototype.validateAccessToken = function (accessToken, idToken, cb) {
+  var decodedIdToken = this.decode(idToken);
+  // if it has a `name` property, it's an error
+  if (decodedIdToken.name) {
+    return cb(decodedIdToken);
+  }
+  if (supportedAlgs.indexOf(decodedIdToken.header.alg) === -1) {
+    return cb(new error.TokenValidationError('Algorithm ' + decodedIdToken.header.alg +
+      ' is not supported. (Expected algs: [' + supportedAlgs.join(',') + '])'));
+  }
+  var sha256AccessToken = sha256(accessToken);
```

Comment:
It worth putting any checks in if this fails before going through all the next steps?

Reply:
the tests already have invalid data, so I think it's not needed, because those methods are not throwing exceptions with invalid data. wdyt?